src/index.ts

```diff
@@ -0,0 +1,17 @@
+import fastify from "fastify";
+
+const server = fastify();
+
+server.get("/", async (req, rep) => {
+  return rep.send("hello world with fastify");
+});
+
+server.listen(8080, (err, address) => {
+  if (err) {
+    console.error(err);
+
+    process.exit(1);
+  }
```

Comment:
super nitpicky - but do we wanna set up prettier so that we won't get changes from format?

Reply:
@kei95 I just created the task here.
https://github.com/ryonryon/trello-clone/issues/4